tests.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var assert = require('chai').assert;
+var combyne = require('combyne');
+var visitCombyne = require('./index');
+
+describe('visit-combyne', function() {
+  var template;
+
+  describe('if-elseif conditionals', function() {
+    beforeEach(function() {
+      template = combyne('{%if case1%}1{%elsif case2%}2{%else%}3{%endif%}');
+    });
+
+    it('correctly recurses over all nodes in conditional chains', function() {
+      var visitedNodes = {};
+      // Count how many nodes of each type are visited
+      visitCombyne(template.tree.nodes, function(node) {
+        if ( ! visitedNodes[node.type] ) { visitedNodes[node.type] = 0; }
+        visitedNodes[node.type]++;
+      });
+
+      assert.equal(visitedNodes.ConditionalExpression, 3,
+        'finds three conditional nodes ("if", "elsif", "else")');
+      assert.equal(visitedNodes.Property, 2,
+        'finds two property nodes, "case1" and "case2"');
+      assert.equal(visitedNodes.Text, 3,
+        'finds three text nodes, "1", "2", "3"');
```

Comment:
Went with `assert` because it's the form you use in Combyne and it can give more informative errors on the command line; I wish `expect` supported per-assertion messages ;)